src/mimefactory.rs

```diff
@@ -316,15 +316,27 @@ impl<'a, 'b> MimeFactory<'a, 'b> {
                     };
                     format!("{}{}", re, chat.name)
                 } else {
-                    let raw = message::get_summarytext_by_raw(
-                        self.msg.viewtype,
-                        self.msg.text.as_ref(),
-                        &self.msg.param,
-                        32,
-                        self.context,
-                    );
-                    let raw_subject = raw.lines().next().unwrap_or_default();
-                    format!("Chat: {}", raw_subject)
+                    match chat.param.get(Param::LastSubject) {
+                        Some(last_subject) => {
+                            let subject_start =
+                                match last_subject[..5].chars().position(|c| c == ':') {
```

Comment:
What makes 5 so special here to only look in the first 5 characters?  How easily will an other language break this?

Reply:
Ah yes, this needs a comment. "Antw:" was the longest abbreviation in https://en.wikipedia.org/wiki/List_of_email_subject_abbreviations#Abbreviations_in_other_languages; that's where the 5 comes from. 

I am ready to discuss this but the number must not be too large not to break messages whose subject contains a ":" in the middle.

Reply:
That's nice, glad I asked.  TIL :)

Reply:
(a comment would be great though)